android/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.benjaminabel.vibration">
-    <uses-sdk
```

Comment:
Removed this since it is always overridden by the value specified in the Gradle build script